--- exiftags-1.01/makers.h.orig	2004-09-15 23:35:55.000000000 +0000
+++ exiftags-1.01/makers.h	2026-05-21 00:00:00.000000000 +0000
@@ -52,8 +52,8 @@
 struct makerfun {
 	int val;
 	const char *name;
-	void (*propfun)();		/* Function to parse properties. */
-	struct ifd *(*ifdfun)();	/* Function to read IFD. */
+	void (*propfun)(struct exifprop *prop, struct exiftags *t);
+	struct ifd *(*ifdfun)(u_int32_t offset, struct tiffmeta *md);
 };
 extern struct makerfun makers[];

--- exiftags-1.01/canon.c.orig	2005-01-05 00:30:32.000000000 +0000
+++ exiftags-1.01/canon.c	2026-05-21 00:00:00.000000000 +0000
@@ -1104,7 +1104,8 @@
  */
 static int
 canon_subval(struct exifprop *prop, struct exiftags *t,
-    struct exiftag *subtags, int (*valfun)())
+    struct exiftag *subtags,
+    int (*valfun)(struct exifprop *aprop, struct exifprop *prop, unsigned char *off, struct exiftags *t))
 {
 	int i, j;
 	u_int16_t v;
